content/en/blog/falcosidekick-ui-2-2-0/index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Falcosidekick-UI 2.2.0
+linktitle: Falcosidekick-UI 2.2.0
+date: 2023-09-14
+author: Thomas Labarussias
+slug: falcosidekick-ui-2-2-0
+images:
+  - /blog/falcosidekick-ui-2-2-0/images/featured.png
+tags: ["Falcosidekick-UI","Release"]
+---
+
+Not so along, we proudly released a new fantastic release of [falcosidekick](/blog/falcosidekick-2-28-0/), it's time for its little brother, [falcosidekick-ui](https://github.com/falcosecurity/falcosidekick-ui) to know the same, with the version v2.2.0.
+
+Let's take a tour to introduce the most important cool new features of this release.
+
+## Disabling the authentication
+
+The previous version introduced a basic auth mechanism to protect the access to the dashboard and API. Some complained it broke the access through their reverse proxy. You can now disable the authentation:
+
+```shell
+-d boolean
+      Disable authentication (environment "FALCOSIDEKICK_UI_DISABLEAUTH")
+```
+
+## Dialog box to display the details of an event
+
+To have a better view of each event, you can now open a dialog box which displays all details but also the raw json of the event. You can event copy it in your clipboard by a simple click.
+
+![dialog-box](/blog/falcosidekick-ui-2-2-0/images/dialog-box.png)
+
+To display the dialog box, just click on the `{...}` at the end of the event row.
+
+![dialog-box-button](/blog/falcosidekick-ui-2-2-0/images/dialog-box-button.png)
+
+## Export
+
+A new `Export` button appeared, it allows you to export all the events found in json format. It takes in consideration the filters, of course.
+
+![export](/blog/falcosidekick-ui-2-2-0/images/export.png)
+
+![json](/blog/falcosidekick-ui-2-2-0/images/json.png)
+
+## Units for TTL
+
+For users with a lot of events, it can be useful to specify a TTL (time to live) for the keys in Redis (the storage backend). It can be done with `-t` argument for a while, and the value had be in seconds, not convenient for long term storages. You can now specify a unit (`s`econds, `m`inutes, `h`ours, `W`eeks, `M`onths, `y`ear). If no unit is specified, it's considered as seconds to avoid to break previous configs.
```

Comment:
```suggestion
For users with a lot of events, it can be useful to specify a TTL (time to live) for the keys in Redis (the storage backend). It can be done with `-t` argument for a while, and the value had to be in seconds, which is not convenient for long-term storage. You can now specify a unit (`s`econds, `m`inutes, `h`ours, `W`eeks, `M`onths, `y`ear). If no unit is specified, it's considered as seconds to avoid breaking previous configs.
```